pkg/limayaml/defaults.go

```diff
@@ -527,6 +532,47 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 	y.CACertificates.Certs = caCerts
 }
 
+func processGuest(format string) (bytes.Buffer, error) {
```

Comment:
I find `process` kind of meaningless, as it doesn't really tell me what is happening.

Maybe `expandGuestTemplate` (and `expandHostTemplate`) are better names?

Reply:
Those names sound better, I was struggling a bit

Reply:
Or even executeGuestTemplate, like template.Execute